tracer/src/Datadog.Trace/Configuration/ImmutableAzureAppServiceSettings.cs

```diff
@@ -172,9 +167,13 @@ private static bool ShouldSkipClientSpanWithinFunctions(Scope? scope)
             return resourceId;
         }
 
-        private string? GetSubscriptionId(IConfigurationSource source)
+        private string? GetSubscriptionId(IConfigurationSource source, IConfigurationTelemetry telemetry)
         {
-            var websiteOwner = source.GetString(ConfigurationKeys.AzureAppService.WebsiteOwnerNameKey);
+            var websiteOwner = new ConfigurationBuilder(source, telemetry)
+                              .WithKeys(ConfigurationKeys.AzureAppService.WebsiteOwnerNameKey)
+                              .AsString()
+                              .Get(websiteOwner => websiteOwner.IndexOf('+') > 0);
+
             if (!string.IsNullOrWhiteSpace(websiteOwner))
```

Comment:
Maybe you should move this logic in the getter?

Reply:
I was undecided tbh 🤷‍♂️ could do.